spectrum/checks.py

```diff
@@ -867,11 +867,18 @@ def _build_url(path, host):
     'articles/{id}/elife-{id}-v{version}.pdf',
     'articles/{id}/elife-{id}-v{version}.pdf'
 )
-PACKAGING_BUCKET = BucketFileCheck(
+PACKAGING_BUCKET_OUTBOX = BucketFileCheck(
     aws.S3,
     SETTINGS['bucket_packaging'],
-    '{vendor}/{folder}/elife{id}.xml',
-    '{vendor}/{folder}/elife{id}.xml'
+    '{vendor}/outbox/elife{id}.xml',
+    '{vendor}/outbox/elife{id}.xml'
```

Comment:
This spectrum code is all new to me. Should / could this last argument, which is maybe the S3 prefix, be ``'{vendor}/outbox/'``?

Reply:
The last argument is a prefix used to cut down on the number of files listed from S3 when the main expression has to be matched. `{vendor}` and `{id}` are substituted when `of()` is called, so in this case we can essentially use the whole file name as the prefix, which is very efficient. In other cases we don't know the file name in advance, so we have to stop the prefix at the "folder" level.

Reply:
Ah yes, the published folder subfolders vary by the date it is run, gotcha - thanks!